misc/exec-command-agent-test/linux.dockerfile

```diff
@@ -15,9 +15,13 @@ MAINTAINER Amazon Web Services, Inc.
 
 WORKDIR /go/src/sleep
 ADD ./sleep /go/src/sleep
-RUN go get -d -v ./...
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -installsuffix cgo -a -o /go/bin/sleep .
+RUN CGO_ENABLED=0 go build -tags integration -installsuffix cgo -a -o /go/bin/sleep .
+
+WORKDIR /go/src/kill
+ADD ./kill /go/src/kill
```

Comment:
just curious, where is this used?

Reply:
This is used in the actual integ test in order to kill the fake ssm agent

Reply:
found it, the actual test code was folded =)